bin/serve.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import { argv } from 'yargs';
+import { existsSync, readFileSync } from 'fs'
+import { TestxServer } from '../src';
+
+async function startServer(model: string) {
+    const server = new TestxServer({
```

Comment:
Is it possible to assign a port instead of having one randomly generated?

Reply:
+100

Reply:
Ok. This will mean refactoring of the testx as it was designed to bring random port :(

Reply:
The server start() method should be already abel to accpet a custom port, and if not is very easy to do it

Reply:
@wtrocki if you can take do something like `port = argv.port || '4001'` and then call it a day

Reply:
The TestxServer start is calling the GraphbackServer start function which accepts a fixed port, is just a matter of adding the argument to the TestxServer start and pass it through 
https://github.com/aerogear/graphql-testx/blob/db2a503e4a1ddf768d18b8eb888e75d6409a9e57/src/TestxServer.ts#L101
https://github.com/aerogear/graphql-testx/blob/db2a503e4a1ddf768d18b8eb888e75d6409a9e57/src/GraphbackServer.ts#L32